assets/js/index.js

```diff
@@ -0,0 +1,351 @@
+      "Eric Patrick Clapton",
+      "Gordon Sumner",
+      "Vincent Damon Furnier",
+    ],
+    correctAnswer: "Vincent Damon Furnier",
+  },
+  {
+    question: "The 1991 album Nevermind is associated with which rock group?",
+    answers: [
+      "Grateful Dead",
+      "Nirvana",
+      "The Ramones",
+      "The Velvet Underground",
+    ],
+    correctAnswer: "Nirvana",
+  },
+];
+const highScoreBtn = document.querySelector("#highscore-link");
+const startBtn = document.querySelector("#start-btn");
+const startContainer = document.querySelector("#start-container");
+const mainContainer = document.querySelector("#main");
+const questionContainer = document.querySelector("#question-container");
+const scoreContainerDiv = document.querySelector("#score-container");
+
+let timer = null;
+
+let timeRemaining = 91;
+
+let currentQuestionIndex = 0;
+
+const initializeLocalStorage = function (key, defaultValue) {
+  //initialize LS
+  const highScore = JSON.stringify(localStorage.getItem(key));
+
+  if (!highScore) {
+    localStorage.setItem(key, JSON.stringify(defaultValue));
+  }
+};
+
+const getFromLocalStorage = function (key, defaultValue) {
+  const localStorageData = JSON.parse(localStorage.getItem(key));
+
+  if (!localStorageData) {
+    return defaultValue;
+  } else {
+    return localStorageData;
+  }
+};
+
+const addToHighScore = function () {
+  //Initialize LS
+  initializeLocalStorage("highScore", []);
+
+  //Get value from textInput
+  const initialsInput = document.getElementById("text-input").value;
+
+  //Construct an object {initials:"", score:0}
+  const scoreObject = { initials: initialsInput, score: timeRemaining };
+
+  //add to LS
+  const highScore = getFromLocalStorage("highScore", []);
+
+  highScore.push(scoreObject);
+
+  localStorage.setItem("highScore", JSON.stringify(highScore));
+};
+
+const renderScoreContainer = function () {
+  //Remove questionContainerDiv
+  questionContainerDiv.remove();
+
+  //Render scoreContainerDiv
+  const scoreContainerDiv = document.createElement("div");
+  scoreContainerDiv.setAttribute("class", "score-container");
+  scoreContainerDiv.setAttribute("id", "score-container");
+
+  //create the h2
+  const endingText = document.createElement("h2");
+  endingText.textContent = "All Done!!!";
+
+  //create the p
+  const finalScoreText = document.createElement("p");
+  finalScoreText.textContent = "Your final score is ";
+
+  //create the span
+  const scoreSpan = document.createElement("span");
+  scoreSpan.textContent = timeRemaining;
+
+  //append the span into the p
+  finalScoreText.appendChild(scoreSpan);
+
+  //create the form
+  const scoreForm = document.createElement("form");
+  scoreForm.setAttribute("class", "score-form");
+
+  //create the label
+  const initialsLabel = document.createElement("label");
+  initialsLabel.setAttribute("for", "initials");
+  initialsLabel.setAttribute("class", "initials");
+  initialsLabel.textContent = "Enter your initials here";
+
```

Comment:
Some error handling or alert would be great here so the user cannot submit a score without a name.

Reply:
done!!!